jinahub/encoders/text/TransformerSentenceEncoder/tests/unit/test_executor.py

```diff
@@ -80,7 +80,7 @@ def test_traversal_path(
 
     basic_encoder.encode(docs=docs, parameters={'traversal_paths': traversal_paths})
     for path, count in counts:
-        embeddings = docs.traverse_flat([path]).get_attributes('embedding')
+        embeddings = DocumentArray(docs.traverse_flat(path)).embeddings
```

Comment:
traverse_flat is not good, u should do `docs[path]`

Reply:
yes, you r right. I missed this part